docs/pages/machine-id/guides/databases.mdx

```diff
@@ -0,0 +1,284 @@
+storage:
+  directory: /var/lib/teleport/bot
+destinations:
+  - directory: /opt/machine-id
+
+    database:
+      service: example-server
+      username: alice
+      database: example
+
+    # If using MongoDB, be sure to include the Mongo-formatted certificates:
+    configs:
+      - mongo
+```
+</Details>
+<Details scope={["oss","enterprise"]} scopeOnly={true}>
+```yaml
+auth_server: "auth.example.com:3025"
+onboarding:
+  join_method: "token"
+  token: "00000000000000000000000000000000"
+  ca_pins:
+  - "sha256:1111111111111111111111111111111111111111111111111111111111111111"
+storage:
+  directory: /var/lib/teleport/bot
+destinations:
+  - directory: /opt/machine-id
+    
+    database:
+      service: example-server
+      username: alice
+      database: example
+    
+    # If using MongoDB, be sure to include the Mongo-formatted certificates:
+    configs:
+      - mongo
+```
+</Details>
+
+Be sure to configure the `token` and `ca_pins` fields to match the output from
+`tctl bots add ...`.
+
+Machine ID also allows you to use Linux ACLs to control access to certificates
+on disk. You will use this to ensure only your application has access to the
+short-lived certificates Machine ID uses.
+
+We'll work with the assumption you will be running Machine ID as the Linux user
+`teleport` and your application as the Linux user `app`. Create and initialize
+the destination directory by running this `tbot init` command either as `root`
+or as the `teleport` user:
+
+```code
+$ tbot init \
+    -c /etc/tbot.yaml \
+    --init-dir=/opt/machine-id \
+    --bot-user=teleport \
+    --owner=teleport:teleport \
+    --reader-user=app
+```
+
+<Admonition type="note" title="`tbot init` Note">
+Be sure to re-run `tbot init ...` as shown here if config templates are added
+or removed from `tbot.yaml`. You may run into permissions errors if `tbot init`
+is not run for new files.
+</Admonition>
+
+Next, you will use systemd to run Machine ID in the background on your
+application node. Create a systemd.unit file at
+`/etc/systemd/system/machine-id.service`:
+
+```systemd
+[Unit]
+Description=Teleport Machine ID Service
+After=network.target
+
+[Service]
+Type=simple
+User=teleport
+Group=teleport
+Restart=on-failure
+ExecStart=/usr/local/bin/tbot start -c /etc/tbot.yaml
+ExecReload=/bin/kill -HUP $MAINPID
+PIDFile=/run/machine-id.pid
+LimitNOFILE=8192
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Additionally, we'll need to create a secondary service to manage the database
+proxy. Create another unit file at `/etc/systemd/system/machine-id-proxy.service`:
+
+```systemd
+[Unit]
+Description=Teleport Machine ID Proxy Service
+After=network.target
+Requires=machine-id.service
+
+[Service]
+Type=simple
+User=teleport
+Group=teleport
+Restart=on-failure
+ExecStart=/usr/local/bin/tbot -c /etc/tbot.yaml proxy --proxy=proxy.example.com:3080 --destination-dir=/opt/machine-id db --port=12345 example-server
```

Comment:
Should this be `tbot start`?

Reply:
Nope, this is an additional service to run the proxy in the background, so we want to run `tbot proxy db ...` (with a bunch of flags in admittedly ugly positions since we have 2 levels of goofy Kingpin argument parsing to contend with)